Readme.md

```diff
@@ -30,10 +34,12 @@ After adding an item, or any other action that update the existing forms, this f
 
 # Initialization
 
+To initialize the project in your cht project, run:
 ```shell
-cht-stock-monitoring-workflow init
+npx cht-stock-monitoring-workflow init
 ```
-  Initialize the project and create a file with name `stock-monitoring.config.json`. This add by default the stock count form empty and update the cht app messages.
+This creates a file with name `stock-monitoring.config.json`, generate a default configurations for the stock count form and updates CHT app messages.
```

Comment:
What if you reformulate the description like:
`This generates a default configurations for the stock count form in a config file named `stock-monitoring.config.json`, and updates CHT app messages.`